sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormField.cs

```diff
@@ -4,35 +4,36 @@
 using System;
 using System.Collections.Generic;
 using System.Globalization;
-using System.Linq;
+using System.Text.RegularExpressions;
 
 namespace Azure.AI.FormRecognizer.Models
 {
     /// <summary>
     /// </summary>
     public class FormField
     {
-#pragma warning disable CA1801
-        internal FormField(KeyValuePair_internal field, ReadResult_internal readResult)
+        internal FormField(string name, int pageNumber, KeyValuePair_internal field, IReadOnlyList<ReadResult_internal> readResults)
         {
-#pragma warning restore CA1801
-            //Confidence = field.Confidence;
-
-            //Name = field.Key.Text;
-            //NameBoundingBox = new BoundingBox(field.Key.BoundingBox);
+            Confidence = field.Confidence;
+            Name = name;
 
-            //if (field.Key.Elements != null)
-            //{
-            //    NameTextElements = ConvertTextReferences(readResult, field.Key.Elements);
-            //}
+            BoundingBox labelBoundingBox = field.Key.BoundingBox == null ? default : new BoundingBox(field.Key.BoundingBox);
+            IReadOnlyList<FormContent> labelFormContent = default;
+            if (field.Key.Elements != null)
+            {
+                labelFormContent = ConvertTextReferences(field.Key.Elements, readResults);
+            }
+            LabelText = new FieldText(field.Key.Text, pageNumber, labelBoundingBox, labelFormContent);
 
-            //Value = field.Value.Text;
-            //ValueBoundingBox = new BoundingBox(field.Value.BoundingBox);
+            BoundingBox valueBoundingBox = field.Value.BoundingBox == null ? default : new BoundingBox(field.Value.BoundingBox);
+            IReadOnlyList<FormContent> valueFormContent = default;
+            if (field.Value.Elements != null)
+            {
+                valueFormContent = ConvertTextReferences(field.Value.Elements, readResults);
+            }
+            ValueText = new FieldText(field.Value.Text, pageNumber, valueBoundingBox, valueFormContent);
 
-            //if (field.Value.Elements != null)
-            //{
-            //    ValueTextElements = ConvertTextReferences(readResult, field.Value.Elements);
-            //}
+            Value = new FieldValue(new FieldValue_internal(FieldValueType.StringType, field.Value.Text, null, null, null, null, null, null, null, field.Value.Text, null, null, null, pageNumber), readResults);
```

Comment:
would it make sense to create an internal constructor for FieldValue that only takes the parameters we need here?
If not, another option could be to put name parameters, otherwise is hard to read and prone to error if someone has to modify it

Reply:
Will fix.